pkg/cmd/pr/merge/merge.go

```diff
@@ -431,7 +431,12 @@ func (m *mergeContext) deleteLocalBranch() error {
 		return fmt.Errorf("failed to delete local branch %s: %w", m.cs.Cyan(m.pr.HeadRefName), err)
 	}
 
-	return nil
+	branch := ""
+	if m.switchedToBranch != "" {
```

Comment:
`m.switchedToBranch` no longer needs to get passed between functions and can just be made a local variable instead of part of `mergeContext`.